### ISSUE TITLE
[BUG] Chips Can't be Permanently Removed From Inventory

### ISSUE THREAD

**Issue by Kadzar** [tsunamayo/Starship-EVO#2778]
### Short Description of the issue:

Although it's possible to drag chips out of the inventory menu, the removed chips return when the menu is reopened.

### Steps to reproduce the issue:

Open Upgrade menu with Chips inside, remove Chips, close menu, reopen menu, see the same Chips return.

### Environment
- Starship EVO version: 20w33a
- Graphics Card: NVIDIA GeForce GTX 1080

### Additional information:

**Comment by tsunamayo** [comment]
Yes it is still wip, I will work on proper ship inventory, but it is gonna be a bit different with cargo crates!
